hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -249,7 +249,7 @@ public void refreshTimeline() throws IOException {
       }
     } else {
       this.commitTimelineOpt = Option.empty();
-      String partitionColumns = HoodieWriterUtils.getPartitionColumns(keyGenerator);
+      String partitionColumns = HoodieSparkWriterUtils.getPartitionColumnsFromKeyGenerator(keyGenerator, props);
```

Comment:
such a long name. :) lets just keep it short.

Reply:
Fixed.